### ISSUE TITLE
MSI Installer on Windows Server 2019

### ISSUE THREAD

**Issue by jblake1991** [cloudbase/ceph-windows-installer#28]
Hello.

After attempting to install the Ceph Pacific and Quincy for Windows x64 on Windows Server 2019 using the MSI, which is on a VMware system, the server no longer boots correctly after a reboot. Has anyone else seen this issue?

**Comment by petrutlucian94** [comment]
Hi, does it enter a crash loop? If so, we'd need a crash dump or stack trace in order to investigate the issue.

**Comment by petrutlucian94** [comment]
I've just double checked the MSIs on WS 2019. You're most probably hitting this issue because of secure boot. Our driver hasn't been signed by Microsoft yet, which is why secure boot has to be disabled.

For what is worth, we're in the process of certifying the driver.

**Comment by jblake1991** [comment]
Thanks, Lucian. Let me double check whether I enabled secure boot. I 
don't believe I enabled this by default.

Kind regards,
Josh


------ Original Message ------
From "Lucian Petrut" ***@***.***>
To "cloudbase/ceph-windows-installer" 
***@***.***>
Cc "jblake1991" ***@***.***>; "Author" 
***@***.***>
Date 13/01/2023 15:56:11
Subject Re: [cloudbase/ceph-windows-installer] MSI Installer on Windows 
Server 2019 (Issue #28)

**Comment by jblake1991** [comment]
Looks like secure boot is enabled by default, I'll try again without it 
enabled. Thanks!

Kind regards,
Josh


------ Original Message ------
From "Lucian Petrut" ***@***.***>
To "cloudbase/ceph-windows-installer" 
***@***.***>
Cc "jblake1991" ***@***.***>; "Author" 
***@***.***>
Date 13/01/2023 15:56:11
Subject Re: [cloudbase/ceph-windows-installer] MSI Installer on Windows 
Server 2019 (Issue #28)

**Comment by petrutlucian94** [comment]
I assume this was caused by secure boot, so I'll close this issue. We're tracking secure boot support here: https://github.com/cloudbase/wnbd/issues/89.

I really wish we were able to provide a Microsoft signed driver. We passed all the WHQL driver attestation tests, however we aren't able to publish the results to Microsoft. Their WHQL result submission platform is broken and they haven't sorted out the problem in more than a year.
